src/Compilers/Test/Core/Generated.targets

```diff
@@ -156,8 +156,8 @@
           <LogicalName>netstandard10.microsoftcsharp.Microsoft.CSharp.dll</LogicalName>
```

Comment:
Looks like this is an old left over?

Reply:
Actually all changes here are wrong, this is a generated file. This isn't us targeting `netstandard1.3` in our builds but rather testing the compilers ability to do builds against `netstandard1.3`